ui/page/root/home_page.go

```diff
@@ -131,15 +131,15 @@ func (hp *HomePage) initPageItems() {
 	}
 
 	// Add trade tab only if not on macOS
-	if !appos.Current().IsDarwin() {
-		// Determine the insertion point, which is second to last position
-		insertionPoint := len(navigationTabTitles) - 1
-		if insertionPoint < 0 {
-			insertionPoint = 0
-		}
-		// Append at the second to last position
-		navigationTabTitles = append(navigationTabTitles[:insertionPoint], append([]string{values.String(values.StrTrade)}, navigationTabTitles[insertionPoint:]...)...)
+	// if !appos.Current().IsDarwin() {
```

Comment:
why is this commented out?

Reply:
sorry, I forgot revert this code when testing this pull request, I will uncomment it.